README.rst

```diff
@@ -185,6 +197,35 @@ are copied. Be aware that system property *is_public* must not be included in
 ``metadata_set``, because it is not a user property but a system one. Anyway,
 *is_public* is unconditionally synchronised.
 
+How the obsoleted images are managed
+------------------------------------
+
+An obsolete image is an image with the *_obsolete* suffix. When an image is
+marked as obsoleted is not synchronised anymore and therefore it is not upload to
+regions where it is not present. However, if an image exists in the remote region
+with the same name but without the suffix, it is renamed and the visibility is
+updated with the value on the master region. Also the properties specified
+in *obsolete_syncprop*, if any, are synchronised. The synchronisation of the
+properties and the visibility is also managed when there is a image in the
+region to synchronise that is already renamed but without the other changes
+propagated.
+
+Actually, there are some checks to do before propagating the changes of an
```

Comment:
My idea in Spanish was "realmente". But I'll remove "actually," I have checked that the sense I had though is not quite correct.